testing/pom.xml

```diff
@@ -247,8 +248,37 @@
           <includes>
             <include>**/*.java</include>
           </includes>
+          <argLine>${failsafe.jacoco.args}</argLine>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <executions>
+          <!--
+          "report" goal can't cross boundaries of modules,
+          while "report-aggregate" can, so let's use it, however
+          by default it will load "jacoco.exec" from this module and from module "jar",
+          so let's also change file name for this module to avoid intersection
```

Comment:
Why do we have to do this in `testing` but not in other modules?

Reply:
In order to aggregate the integration tests run in this module.